.github/workflows/deploy.yaml

```diff
@@ -283,6 +283,16 @@ jobs:
 
       - uses: ./.github/actions/gradle-setup
 
+      - name: 'Generate Gaia-X Self-Description'
```

Comment:
why introduce jinja2 instead of using Terraform to generate the document as we do for others?
This is also slow (15s)